plans/main.fmf

```diff
@@ -0,0 +1,5 @@
+/system_test:
```

Comment:
Non-blocking:

Would it be convenient, and idiomatic enough, to put this somewhere where it doesn’t have its own top-level directory?

maybe inside the existing `systemtest`? Or possibly somewhere in `hack` or `contrib`?

Reply:
tmt-enabled repos usually put the test plans in `plans` dir, so I'm hoping to go with that. If it's any consolation, if we get to the point where we don't have to depend on cirrus, maybe we can remove the contrib dir.

Reply:
WFM, let’s keep using the idiomatic location.